queue_services/strr-pay/devops/gcp/clouddeploy.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: deploy.cloud.google.com/v1
+kind: DeliveryPipeline
+metadata:
+ name: strr-pay-pipeline
+description: Deployment pipeline
+serialPipeline:
+ stages:
+ - targetId: bcrbk9-dev
+   profiles: [dev]
+   strategy:
+    standard:
+      verify: false
+   deployParameters:
+   - values:
+      deploy-env: "development"
+      deploy-project-id: "bcrbk9-dev"
+      service-name: "strr-pay-dev"
+      container-name: "strr-pay-dev"
+      cloudsql-instances: "bcrbk9-dev:northamerica-northeast1:strr-db-dev"
+      service-account: "sa-api@bcrbk9-dev.iam.gserviceaccount.com"
+
+ - targetId: bcrbk9-test
+   profiles: [test]
+   strategy:
+    standard:
+      verify: false
+   deployParameters:
+   - values:
+      deploy-env: "test"
```

Comment:
as i mentioned in the comment for renaming testing to test, you need to use an environment name that has TESTING = False. So, you have 2 choices, use 'development' here or create another config environment named 'test' (renaming current 'test' to 'unittest'), and using that instead.